src/docs/design/design-audit.mdx

```diff
@@ -0,0 +1,212 @@
+
+## Communicate with User Journey
+
+User experience design is centered around users. Instead of talking about the technical nature of your infrastructure, teams should explain to auditors in terms of user journeys. Who will be using the application? What do they care about the most? How will the application solve this user’s problem?
+
+## Communicate the Limit
+
+If the product has been around for a long time, the team should mention to its auditor any legacy issues. Auditors should also be informed of other limitations, such as limited ethernet connection speed, mobile app-exclusive features, etc.
+
+## Define a Proper Scope
+
+One reason we recommend doing design audits periodically is so that product teams do not work alone for too long before they receive any external feedback. This also helps keep the audit scope focused on a few workflows or screens at a time.
+
+If the product is complex or has never been audited, consider arranging only a few key workflows or screens to be audited. A large scope will “dilute” auditors’ attention and tend to contain a lot of repetitive comments. The product team should select the part of the product that is either most important to its users, or that can help guide future redesign efforts for other pages.
+
+# Issue Severity
+
+Design audits identify issues of different severities. Severity ratings are determined by their impact on the users. Brightlayer UI recommends using the following severity rating.
+
+A **top severity** issue is a design flaw that prevents a user from completing a certain task.
+
+<ImageGrid images={[TopSeverity]} regularWidth />
+
+A **high severity** issue greatly confuses or blocks its users, although eventually they might learn to tolerate it.
+
+<ImageGrid images={[HighSeverity]} regularWidth />
+
+A **medium severity** issue makes users hesitate, but they are usually still able to complete their task after some guesswork.
+
+<ImageGrid images={[MediumSeverity]} regularWidth />
+
+A **low severity** issue is cosmetic and does not interfere with functionality or usability.
+
+<ImageGrid images={[LowSeverity]} regularWidth />
+
+In addition, auditors may suggest **features** as well. Features are good-to-have improvements; users often notice them in other Brightlayer UI apps.
+
+<ImageGrid images={[Feature]} regularWidth />
+
+As hinted above, audit notes may also be tagged with whether an issue is about usability or branding. This further helps product teams think about usability systematically.
+
+Auditors are recommended to color code their design audit notes based on severity.
+
+# How to Conduct Design Audits (For Auditors)
+
+## Prepare Deliverables
+
+A full audit done by the Brightlayer UI team typically includes two parts: detailed notes that capture everything, and a presentation slide deck including only items worth discussing.
```

Comment:
```suggestion
A full audit done by the Brightlayer UI team typically includes two parts: detailed notes that capture everything, and a presentation slide deck including only items that merit further discussion.
```

Reply:
this further complicate the sentence

Reply:
maybe 'worth discussing together'. I think I was trying to avoid the implication that all the other stuff is 'not worth discussing'